### ISSUE TITLE
Integration Test - Edits does not update names various tabs. 

### ISSUE THREAD

**Issue by AlagappanRa** [AlagappanRa/ped#8]
Student 1:

![image.png](https://raw.githubusercontent.com/AlagappanRa/ped/main/files/80d2f2ee-4279-453b-888d-535e58e87de8.png)

Student 2:

![image.png](https://raw.githubusercontent.com/AlagappanRa/ped/main/files/f47952ca-f92c-4b81-b6f8-69317c284979.png)

Consulation set up using createsession command. 

![image.png](https://raw.githubusercontent.com/AlagappanRa/ped/main/files/ea9a5a70-c9bb-480f-80d7-e617fff3e383.png)

Now, I proceed to edit the name of student 1 because i realise i mispelt his name so i change it to alexteo 

![image.png](https://raw.githubusercontent.com/AlagappanRa/ped/main/files/221cec2c-cd53-482a-a8cc-ed496d6ebb5c.png)

I end up getting this as the output.

![image.png](https://raw.githubusercontent.com/AlagappanRa/ped/main/files/302049e2-c3fb-4ca4-a63c-74452b99f5df.png)

Now when i scroll over to the consultation tab using tab 5

![image.png](https://raw.githubusercontent.com/AlagappanRa/ped/main/files/001c9309-5bc2-4c96-988d-81227e225e3d.png)

The consultation session between alexteo (after re-naming) and disheng still shows up as alexyeoh and disheng under the tags.

Same issue with sessions tabs.

![image.png](https://raw.githubusercontent.com/AlagappanRa/ped/main/files/9a174f1e-8971-455a-8f83-d479cd8894e6.png)

TLDR:
Name does not update under consultations > tags and under sessions > tags when the name edit is made under students tab.
